tests/sentry/web/frontend/test_organization_auth_settings.py

```diff
@@ -258,3 +258,120 @@ def test_superuser_disable_provider(self, email_unlink_notifications):
         assert not om.user.is_managed
 
         assert email_unlink_notifications.delay.called
+
+    def test_edit_sso_settings(self):
+        organization, auth_provider = self.create_org_and_auth_provider()
+        self.create_om_and_link_sso(organization)
+        path = reverse('sentry-organization-auth-provider-settings', args=[organization.slug])
+
+        assert not getattr(auth_provider.flags, 'allow_unlinked')
+        assert organization.default_role == 'member'
+        self.login_as(self.user, organization_id=organization.id)
+
+        with self.feature('organizations:sso-basic'):
+            resp = self.client.post(path, {
+                'op': 'settings',
+                'require_link': False,
+                'default_role': 'owner',
+            })
+
+        assert resp.status_code == 200
+
+        auth_provider = AuthProvider.objects.get(organization=organization)
+        assert getattr(auth_provider.flags, 'allow_unlinked')
+        assert Organization.objects.filter(id=organization.id, default_role='owner').exists()
```

Comment:
Why do you need to check that the organization exists?

Reply:
Was checking the default_role. Could make that clearer